arctic/store/version_store.py

```diff
@@ -517,6 +522,31 @@ def append(self, symbol, data, metadata=None, prune_previous_version=True, upser
         return VersionedItem(symbol=symbol, library=self._arctic_lib.get_name(), version=version['version'],
                              metadata=version.pop('metadata', None), data=None)
 
+    @mongo_retry
+    def update(self, symbol, data, **kwargs):
```

Comment:
See previous comment about this. I can add a new field to the segment docs that maps to the version number in the version doc.